esti/s3_gateway_test.go

```diff
@@ -338,17 +375,10 @@ func TestS3CopyObjectMultipart(t *testing.T) {
 	destRepo := createRepositoryByName(ctx, t, destRepoName)
 	defer deleteRepositoryIfAskedTo(ctx, destRepoName)
 
-	// content
-	r := rand.New(rand.NewSource(17))
-	objContent := testutil.NewRandomReader(r, largeDataContentLength)
-	srcPath := gatewayTestPrefix + "source-file"
-	destPath := gatewayTestPrefix + "dest-file"
-
-	// upload data
 	s3lakefsClient := newMinioClient(t, credentials.NewStaticV4)
-	_, err := s3lakefsClient.PutObject(ctx, repo, srcPath, objContent, largeDataContentLength,
-		minio.PutObjectOptions{})
-	require.NoError(t, err)
+
+	srcPath, objectLength := getOrCreatePathToLargeObject(t, ctx, s3lakefsClient, repo, branch)
```

Comment:
```suggestion
	srcPath, objectSize := getOrCreatePathToLargeObject(t, ctx, s3lakefsClient, repo, branch)
```

Reply:
Not done: this file consistently uses "length" not "size", for instance [here](https://github.com/treeverse/lakeFS/pull/7609/files#diff-645a01e5b3aac0d2e29d07ef7b6e18d8b5105341944b268480d576228f2e4a1aR455).